src/Representation/Functions/MemorySet.cs

```diff
@@ -36,13 +36,15 @@ public SetMemory(IExpression destination, IExpression value)
             _value = value;
         }
 
-        public void Invoke(IState state, BigInteger value)
+        public void Invoke(IState state, BigInteger bytes)
         {
-            foreach (var offset in Enumerable.Range(0, (int) value))
-            {
-                var address = _destination.Add(state.Space.CreateConstant(_destination.Size, offset));
-                state.Memory.Write(address, _value);
-            }
+            var size = ((Bytes) (uint) bytes).ToBits();
+            var buffer = state.Space.CreateConstant(size, BigInteger.Zero);
```

Comment:
Do you think we should introduce `CreateZero` in the same way we have `ConstantUnsigned.CreateZero` in `Computation`?

Reply:
Leave it for now.

Reply:
Happy to do it on a completely separate PR.